lang/cs/BUILD.md

```diff
@@ -61,6 +63,39 @@ To run .NET tests on local machine from command line, execute
 
 `TestRunner.proj` already has a filter set up to exclude Yarn tests.
 
+***For Building DotNet projects:***
+
+We are in the process of converting the C# support to use dotnet core as well as support Visual Studio 2017. As a result 
+there are *.DotNet.csproj files that sit beside the existing csproj files. To build the dotnet projects, you will need at 
+least the .NET Core SDK installed on your machine. Even though dotnet is supported on both windows and linux, the projects
+will only build fully on windows since they do target .net45. The .NET Core SDK is located on github here:
+
+[https://github.com/dotnet/cli](https://github.com/dotnet/cli)
```

Comment:
Do I need to install this even if I already have VS 2017? Would be good to clarify